util/AllPixelConfigure.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
You don't need this except in Python 2 - and now the codebase is entirely 3 on this branch.